src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: flex-end;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 0;
+}
+
+.header .nav__item {
+  display: flex;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+  text-align: center;
+  padding: 0;
```

Comment:
can be removed, if you need - reset on tag directly

if you need to reset default styles - do it for tags

if you need to add like own stylization => use classes